kiota_abstractions/store/backing_store_parse_node_factory.py

```diff
@@ -21,17 +21,11 @@ def __init__(self, concrete: ParseNodeFactory) -> None:
         """
 
         def func1(x):
-            if isinstance(x, BackedModel):
-                backed_model = x
-                backing_store = backed_model.get_backing_store()
-                if backing_store:
-                    backing_store.set_is_initialization_completed(False)
+            if isinstance(x, BackedModel) and x.backing_store:
```

Comment:
Renaming `func1` and `func2` would help with readability

Reply:
Updated in https://github.com/microsoft/kiota-abstractions-python/pull/101/commits/a0101da454f53f7301119c1377efcc5d4d0a1f30